portfolio/src/main/webapp/script.js

```diff
@@ -44,8 +44,26 @@ function addNavBar(){
 }
 
 function addHello(){
-    fetch('/data').then(response => response.text()).then(response =>
-    response.replace("<h1>", ' '). replace("</h1>", ' ')).then((greeting) =>
-    {document.getElementById('hello-container').innerText = greeting; 
+    fetch('/data').then(response => response.json()).then((arrayString) =>
+    {
+        const helloElement = document.getElementById('hello-container');
+        helloElement.innerHTML = '';
+        helloElement.appendChild(createHeading(arrayString[0]));
+        helloElement.appendChild(createList('First Comment: ' + arrayString[1]));
+        helloElement.appendChild(createList('Second Comment: ' + arrayString[2]));
+        helloElement.appendChild(createList('Third Comment: ' + arrayString[3]));
```

Comment:
Should this be a loop?

Reply:
Hi! I will make this change for the future functions I do - thank you!